server/requirements.txt

```diff
@@ -15,4 +15,4 @@ timm==0.6.13
 typeguard==2.13.3
 accelerate==0.18.0
 pytesseract==0.3.10
-gradio==3.24.1
\ No newline at end of file
+gradio==4.18.0
```

Comment:
**Suggestion**: Consider performing a thorough dependency compatibility check to ensure that the gradio upgrade to version 4.18.0 does not introduce any breaking changes or conflicts with other packages. It's also advisable to review the gradio release notes for version 4.18.0 and any migration guides provided by the maintainers to facilitate a smooth transition. <br> **Code Suggestion**: 
 ```
 +gradio==4.18.0 # Ensure compatibility with other dependencies
 ```